### DISCUSSION TITLE
De-bounce time for digital read/write

### DISCUSSION THREAD

**Discussion by aakashbadiani** [MrYsLab/telemetrix#70] [Ideas]
Not sure if this is already possible or not but a de-bounce time for digital read/write would be a really useful. 

I have a normally closed limit switch that is active low. Every time someone uses the same power rail, it momentarily drops voltage there by showing up as a digital flicker. This is actually fairly common and there is typically a de bounce time programmed in PLCs out in the open world. 

Currently i am doing this within the python side since I cant figure out of this is natively provided. 

Your library is amazing,

**Comment by MrYsLab** [comment]
Thanks for the comment. An [example for the UNO R4](https://github.com/MrYsLab/telemetrix-uno-r4/blob/e1c47ef236bc3f805a430646ffafc5a405074e70/telemetrix_uno_r4/r4_wifi_examples/threaded/WIFI/wwt_digital_input.py#L46) demonstrates adding debounce within the callback method. This should work for the Uno R3 as well.

Please let me know if this accomplishes what you need.
